ci/setup_cmake.sh

```diff
@@ -12,8 +12,16 @@ if [ "x$CMAKE_VERSION" = "x" ]; then
   export CMAKE_VERSION=3.15.2
 fi
 
+# This variable not set on CI pipeline for only legacy environment(GCC 4.8).
+# With 1.13.0 version, C++14 must be set which does not supported by legacy environment anymore.
+# Also with this version, release version path needs to be adapted.
 if [ "x$GOOGLETEST_VERSION" = "x" ]; then
-  export GOOGLETEST_VERSION=1.12.1
+  export GOOGLETEST_VERSION=1.10.0
```

Comment:
Why is default version changed to 1.10.0, can it remain as 1.12.1 ?

Reply:
If GoogleTest version is not set on CI Pipeline, it was set as 1.10.0 by default. It could not be 1.12.1 version because GCC 4.8 oldish environment to support C++14.